### PR TITLE
New Example (fancy?)

### PR DESCRIPTION
**Pull request by jzwar**
# Overview
- Demonstrate more of the `show_options`
- Bragging rights for splinepy
- Fancy graphics

## Addressed issues
...

## Showcase
This is a possible outcome:

1. Isophotes
image

2. Gaussian Curvature
image

3. Mean Curvature
image

For the given spline
image

## Checklists
* [ ] Documentations are up-to-date.
* [X] Added example(s)
* [ ] Added test(s)

### CODE REVIEW

**Comment by j042** [comment]
Fancy

File: examples/show_isophotes_and_curvature.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+a.show_options["knots"] = False
+
+# Show Isophotes
+a.show_options["data_name"] = "isophotes"
+a.show(n_colors=2, resolutions=1000, title="Isophotes")
+
+# Show Gaussian Curvature
+a.show_options["data_name"] = "gaussian_curvature"
+a.show_options["scalarbar"] = True
+a.show(n_colors=2, resolutions=100, title="Gaussian Curvature")
```
Comment (j042) [inline examples/show_isophotes_and_curvature.py:136]: what does `n_colors` do?

File: examples/show_isophotes_and_curvature.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+# Show Curvature
+a.show_options["data_name"] = "mean_curvature"
+a.show_options["scalarbar"] = True
+a.show(n_colors=2, resolutions=100, title="Mean Curvature")
```
Comment (j042) [inline examples/show_isophotes_and_curvature.py:141]: same here

**Review by j042 — APPROVED** [review]
Great, thank you!

File: examples/show_isophotes_and_curvature.py (hunk 1 above)
Comment (jzwar) [reply examples/show_isophotes_and_curvature.py:136]: Didn't have the expected effect. Deleted it

**Pull request merged**